axelrod/strategies/alternator.py

```diff
@@ -17,7 +18,7 @@ class Alternator(Player):
         'manipulates_state': False
     }
 
-    def strategy(self, opponent):
+    def strategy(self, opponent: axelrod.player.Player) -> axelrod.actions.Action:
```

Comment:
Just `Player` is fine here since you imported it; you can import `Action` from `axelrod.actions` and then there's no need to import axelrod in line 2.